frontend/src/pages/Home/components/Criteria/Criteria.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import slugify from 'slugify';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { updateCategory, updateUrl, updateDesigner, updateSort } from './filterSlice';
+import './Criteria.scss';
+
+interface IProps {
+  title: string;
+  criteria: string[];
+  type: string;
+}
+const Criteria: React.FC<IProps> = (props: IProps) => {
+  const { type, title, criteria } = props;
+  const dispatch = useDispatch();
+  const sortList = [
+    {
+      key: 'name',
+      order: 'ASC',
+    },
+    {
+      key: 'createdAt',
+      order: 'DESC',
+    },
+    {
+      key: 'price',
+      order: 'ASC',
+    },
+    {
+      key: 'price',
+      order: 'DESC',
+    },
+  ];
+  const handleClick = (idx: number) => {
```

Comment:
add return type, ex: void. some functions may return something else that's why we need a return type